frontend/src/routes/(authenticated)/user/+page.svelte

```diff
@@ -37,31 +42,97 @@
       );
     }
   }
+
+  let sendingVerificationEmail = false;
+  let sentVerificationEmail = false;
+
+  async function sendVerificationEmail(): Promise<void> {
+    sendingVerificationEmail = true;
+    const result = await fetch('/api/user/sendVerificationEmail', { method: 'POST' });
+    if (!result.ok) throw Error(`Failed to send verification email. ${result.status}: ${result.statusText}.`);
+    sentVerificationEmail = true;
+    sendingVerificationEmail = false;
```

Comment:
if we throw for an error then sending will stay `true`, put this in a finally block?